client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
>but this order isn't very useful.

Taking a step back, we should consider the purpose of `action-types.js`. Is it for discoverability of actions? I'd argue that each sub-trees own `actions.js` is more useful in this context. Rather, I'd argue that `action-types.js` purpose is to prevent type conflicts and have a single reference of all action types in the system.  Furthermore, I'd personally find that developer confusion is more likely to arise from deciding where add a new action type or find an existing action type if we don't enforce a guideline like alphabetization.

Do you have another recommendation in mind that satisfies all of the above? Perhaps grouped action types with a comment describing each group?